src/components/Settings/AuthButtons.vue

```diff
@@ -73,7 +89,6 @@ export default {
 
 span.user-type-note {
   color: var(--settings-text-color);
-  text-transform: capitalize;
```

Comment:
Very minor thing, but when the usernames in basic auth are all lowercase, it looks better with the first letter as a capital, which is what `text-transform: capitalize;` did

Reply:
I removed this because it was changing `my.name@example.com` to `My.Name@Example.Com`.
I am happy to revert this if you prefer. Formatting email usernames can be addressed at another time.

Reply:
I see, that makes perfect sense then. Much better to have a lowercase username than a weirdly capitalized email address